ecl/data_source_ecl_network_fic_gateway_v2_test.go

```diff
@@ -0,0 +1,176 @@
+package ecl
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccNetworkV2FICGatewayDataSource_basic(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skip this test in short mode")
+	}
+
+	resource.Test(t, resource.TestCase{
```

Comment:
並行実行できる場合は以下のように修正してください。
```suggestion
	resource.ParallelTest(t, resource.TestCase{
```